sdk/search/Install-BuildTools.ps1

```diff
@@ -0,0 +1,3 @@
+$scriptDir = Split-Path $MyInvocation.MyCommand.Path -Parent
```

Comment:
@brjohnstmsft any reason to not directly include this in your generate.ps1 script? Do you expect folks to call it independently?

Reply:
Historically everyone would just run msbuild build.proj at the root of the repo once after doing git clean -dxf, so if you never clean, you never need to run it again. The intended usage model of this is the same.

Reply:
This is only setting up the PS modules and thus is only interesting for people that call generate.ps1 (or work directly with the PS cmdlets). For that reason it fells like adding these as a step in the generate.ps1 scripts would help people not have to remember to see if this is setup correctly or not.

Reply:
Sure, I can make that change.